README.md

```diff
@@ -258,9 +258,12 @@ yarn test
 
 To deploy the bridge on testnets:
 
+Firt make sure to initialize the allowed list of aTokens & staticATokens on ./constants/allowlistedTokens.ts
```

Comment:
Not sure to get the meaning of this sentence. Also, there is a typo: `Firt` --> `First`